src/Core/Resolvers/MultipleCreateOrderHelper.cs

```diff
@@ -347,7 +347,8 @@ public class MultipleCreateOrderHelper
             {
                 Tuple<IValueNode?, SyntaxKind> fieldDetails = GraphQLUtils.GetFieldDetails(field.Value, context.Variables);
                 SyntaxKind fieldKind = fieldDetails.Item2;
-                if (GraphQLUtils.IsScalarField(fieldKind) && metadataProvider.TryGetBackingColumn(entityName, field.Name.Value, out string? backingColumnName))
+                if ((GraphQLUtils.IsScalarField(fieldKind) || fieldKind is SyntaxKind.NullValue)
```

Comment:
Can the condition to check `fieldKind is SyntaxKind.NullValue` be part of the implementation of the function `GraphQLUtils.IsScalkarField(fieldKind)` ?

Reply:
NullValue can be for a relationship field as well as for a column field. The value `NULL` does not fit into the definition of either being a complex (relationship) field or a scalar field.